billing.css

```diff
@@ -0,0 +1,158 @@
+/* GLOBAL */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+/* COLORS */
+:root {
+    --black: #000000;
+    --white: #FFFFFF;
+    --green: rgba(134, 198, 160, 0.56);
+    --gray: #F2F2F2;
+    --darkBlue: #0C445B;
+    --darkGray: #333333;
+    --mediumGray: #E0E0E0;
+}
+main {
+    display: flex;
+}
+aside {
+    width: 20vw;
+    padding: 80px 12px 0 0;
+    background-color: var(--darkBlue);
+    height: 100vh;
+}
+#billing-form {
+    width: 70%;
+    height: 29rem;
+    margin: 5rem auto;
+    border: 1px solid var(--mediumGray);
+    border-radius: 12px;
+    padding: 2rem 3rem;
+    font-family: 'Poppins', sans-serif;
+}
+.billing-form-header {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 1.4rem;
+}
+.billing-title {
+    margin-bottom: 0.2rem;
+}
+.billing-sec-title {
+    font-size: 1.1rem;
+    font-weight: 500;
+}
+.account-status {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+}
+.account-status-btn {
+    width: 5rem;
+    height: 1.5rem;
+    background-color: var(--green);
+    color: var(--white);
+    border: none;
+    border-radius: 2px;
+    cursor: pointer;
+    font-family: 'Poppins',sans-serif;
+}
+.input-wrapper {
+    position: relative;
+   
+}
+.account {
+    position: absolute;
+    top: 33%;
+    left: 1rem;
+    transform: translateY(-50%);
+    display: flex;
+    gap: 0.5rem;
+}
+.text-icon img {
+    margin-top: 1.3rem;
+}
+.account-text {
+    font-size: 1rem;
+    font-weight: 600;
+    color: var(--darkBlue);
+    margin: 1.2rem 0;
+}
+.account-input {
+    width: 13rem;
+    height: 4rem;
+    text-align: left;
+    padding: 2rem 0 0.7rem 2.6rem;
+    border: none;
+    border-radius: 4px;
+    box-shadow: 1px 2px 7px -1px rgb(200, 200, 200);
```

Comment:
nice use of variable. but you were not consistent. try to be consistent on design